### PR TITLE
bugfix/v1-070/responsive my courses fixed

### PR DESCRIPTION
**Pull request by Julia-Maliutsina**
Fixed all the issues on common course component, my courses and detailed course page.

### CODE REVIEW

**Review by hauteballe — DISMISSED** [review]

File: frontend/src/components/Layout/MobileSearch/MobileSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+              <AccordionDetails>Statuses here</AccordionDetails>
+            </FilterAccordion>
+            <FilterAccordion>
+              <AccordionSummary expandIcon={<ExpandMoreIcon fontSize="small" />}>
+                <Typography>Technology</Typography>
+              </AccordionSummary>
+              <AccordionDetails>Technologies here</AccordionDetails>
+            </FilterAccordion>
+          </Filter>
+        ) : null}
```
Comment (k-krasilnikova) [inline frontend/src/components/Layout/MobileSearch/MobileSearch.tsx:55]: We shouldn't use `null` in UI!!!

File: frontend/src/pages/MyCourses/MyCoursesContainer.tsx [hunk 2]
```diff
@@ -7,19 +7,42 @@ import MyCoursesList from './MyCoursesList';
...
+  let windowWidth = WINDOW_SIZE.xl.name;
+  if (window.innerWidth < WINDOW_SIZE.xl.width && window.innerWidth > WINDOW_SIZE.lg.width) {
+    windowWidth = WINDOW_SIZE.lg.name;
+  }
+  if (window.innerWidth < WINDOW_SIZE.lg.width && window.innerWidth > WINDOW_SIZE.md.width) {
+    windowWidth = WINDOW_SIZE.md.name;
+  }
+  if (window.innerWidth < WINDOW_SIZE.md.width && window.innerWidth > WINDOW_SIZE.sm.width) {
+    windowWidth = WINDOW_SIZE.sm.name;
+  }
```
Comment (k-krasilnikova) [inline frontend/src/pages/MyCourses/MyCoursesContainer.tsx:35]: Do we really need to duplicate those conditions in 3 components?

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

File: frontend/src/pages/MyCourses/MyCoursesContainer.tsx (hunk 2 above)
Comment (Julia-Maliutsina) [reply frontend/src/pages/MyCourses/MyCoursesContainer.tsx:35]: Removed

File: frontend/src/components/Layout/MobileSearch/MobileSearch.tsx (hunk 1 above)
Comment (Julia-Maliutsina) [reply frontend/src/components/Layout/MobileSearch/MobileSearch.tsx:55]: Fixed

**Review by hauteballe — APPROVED** [review]

**Review by k-krasilnikova — APPROVED** [review]
Good job!

**Review by NikitaKhadnevich — APPROVED** [review]

**Pull request merged**